Source/CarthageKit/Project.swift

```diff
@@ -768,8 +761,9 @@ public final class Project { // swiftlint:disable:this type_body_length
 			}
 			.zip(with: submodulesSignal)
 			.flatMap(.merge) { dependencies, submodulesByPath -> SignalProducer<(), CarthageError> in
+				let limit = UInt(ProcessInfo.processInfo.activeProcessorCount)
 				return SignalProducer<(Dependency, PinnedVersion), CarthageError>(dependencies)
-					.map { (dependency, version) -> SignalProducer<(), CarthageError> in
+					.flatMap(.concurrent(limit: limit)) { (dependency, version) -> SignalProducer<(), CarthageError> in
```

Comment:
I realize that this is functionally equivalent to the existing behavior… but I don't think this is really an appropriate limit since this is a disk- and network-intensive task and not one that's CPU-limited. (Just because you a 16-core processor doesn't mean that your network and disk can handle 16 simultaneous checkouts.)

Could we lower this? Maybe set it to 3 or 4? 😕